subprojects/native/src/main/java/org/gradle/internal/nativeintegration/services/NativeServices.java

```diff
@@ -297,6 +310,11 @@ protected NativeCapabilities createNativeCapabilities() {
             public boolean isNativeIntegrationAvailable() {
                 return useNativeIntegrations;
             }
+
+            @Override
+            public boolean isFileSystemWatchingAvailable() {
+                return useFileSystemWatching;
```

Comment:
Shall we name the local variable and the getter the same?

Reply:
I changed the name of the `getter`.